ppx/ftp.py

```diff
@@ -205,15 +217,21 @@ def _parse_files(self):
 
         return files + new_files, dirs + new_dirs
 
-    def download(self, files, dest_dir, force_=False, silent=False):
+    def download(
+        self,
+        files: Union[str, List[str]],
+        dest_dir: Union[Path, CloudPath],
```

Comment:
We don't have type hints throughout the code currently, so I think we should remove these.